BookOfHabits/BookOfHabitsMicroservice.Domain.Entity/Enums/OptionTemplate.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookOfHabitsMicroservice.Domain.Entity.Enums
+{
+    [Flags]
+    public enum OptionTemplate
+    {
```

Comment:
Use a plural type name for an enumeration with bit fields - https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations